TwipsterApp/Models/Post.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace TwipsterApp.Models
+{
+    public class Post
+    {
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        public int UserId { get; set; }
+        [Required]
+        public DateTime PostTime { get; set; }
+        [Required]
+        [MinLength(3),MaxLength(225)]
+        public string Content { get; set; }
+
+        public User User { get; set; }
```

Comment:
Mark this field as virtual (public virtual User User { get; set; }).

Reply:
Why?

Reply:
Because you are not using it, and It is needed for db scheme, so it will be loaded every time. If you use virtual here EF will know that this variable is used only for db scheme